### DISCUSSION TITLE
Does Deepgram offer diarization for pre-recorded audio files?

### DISCUSSION THREAD

**Discussion by luisedelgado** [deepgram/community#794] [General help] [answered]
I have a local example where I'm using these flags:
```
options = PrerecordedOptions(
    paragraphs=True,
    model="nova-2",
    detect_language=True,
    diarize=True,
)
```
But the output of a pre-recorded conversation between two people comes out as a single paragraph. So I don't think the diarization flag is doing anything in the `prerecorded` codepath. I need the [speaker1], [speaker2] type of transcript. Does this exist with Deepgram?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@luisedelgado  We do, our Docs on using Diarization with Pre-Recorded audio [are here. ](https://developers.deepgram.com/docs/diarization)

There are certain cases where Diarization won't always pick up the unique speakers and the results might come back a single speaker.

Depending on what type of audio you have [MultiChannel](https://developers.deepgram.com/docs/multichannel-vs-diarization) can help aid Diarization in being more successful.
